archive/zip_archiver_test.go

```diff
@@ -29,6 +32,43 @@ func TestZipArchiver_File(t *testing.T) {
 		"test-file.txt": []byte("This is test content"),
 	})
 }
+func TestZipArchiver_FileModified(t *testing.T) {
+	const (
+		zipFilePath = "archive-file.zip"
+		toZipPath   = "./test-fixtures/test-file.txt"
```

Comment:
We need to use `filepath` for all paths. I know there are a tin of broken tests as it is, but I would like to get testing to work on windows someday.